recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set sha256 = "ebc4e2bf8c8266212e342ff8ec4f6ae469e8c41a05d099b6778de8424ce32563" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.patch  # [osx]
```

Comment:
FWIW recent versions of clang 3.7.0+ can be built with OpenMP, but FWICT Apple is disappointingly not doing this and is really quiet about why. We may end up just doing it in our own `clang` compiler. ( https://github.com/conda-forge/clangdev-feedstock/issues/3 )

ref: https://forums.developer.apple.com/thread/18207
ref: https://forums.developer.apple.com/thread/62381
ref: https://forums.developer.apple.com/message/13540